src/js/utils/CRONValidatorUtil.js

```diff
@@ -0,0 +1,214 @@
+
+/**
+ * Validator utilities for CRON syntax
+ */
+class CRONValidatorUtil {
+
+  /**
+   * Test if the given string is a valid CRON syntax
+   *
+   * @param {string} value - The cron string to test
+   * @returns {string} - Returns null if test passes otherwise return the error string
+   */
+  static testCronString(value) {
```

Comment:
We've decided to use plain objects instead of `static` class methods for utils. Let's use a simple object for now and revisit the decision later. I'll create an issue to revisit that decision as would prefer static methods.